test/end-to-end/core.sh

```diff
@@ -326,14 +326,22 @@ os::cmd::expect_success 'oc whoami'
 
 echo "[INFO] Running a CLI command in a container using the service account"
 os::cmd::expect_success 'oc policy add-role-to-user view -z default'
-oc run cli-with-token --attach --image="openshift/origin:${TAG}" --restart=Never -- cli status --loglevel=4 > "${LOG_DIR}/cli-with-token.log" 2>&1
```

Comment:
@kargakis I don't think the assessment is correct, at least not in cases such as this one: https://ci.openshift.redhat.com/jenkins/job/test_pull_requests_origin_integration/7303/console.

```
Running test/end-to-end/core.sh:328: executing 'oc policy add-role-to-user view -z default' expecting success...
SUCCESS after 0.261s: test/end-to-end/core.sh:328: executing 'oc policy add-role-to-user view -z default' expecting success
There was no output from the command.
There was no error output from the command.
Running test/end-to-end/core.sh:330: executing 'cat '/tmp/openshift/test-end-to-end-docker//logs/cli-with-token.log'' expecting success and text 'Using in-cluster configuration'...
FAILURE after 0.014s: test/end-to-end/core.sh:330: executing 'cat '/tmp/openshift/test-end-to-end-docker//logs/cli-with-token.log'' expecting success and text 'Using in-cluster configuration': the output content test failed
Standard output from the command:
Waiting for pod test/cli-with-token to be running, status is Pending, pod ready: false
Waiting for pod test/cli-with-token to be running, status is Pending, pod ready: false
If you don't see a command prompt, try pressing enter.
In project test on server https://172.30.0.1:443

pod/cli-with-token runs openshift/origin:f0d40dd

2 warnings identified, use 'openshift cli status -v' to see details.
Waiting for pod test/cli-with-token to terminate, status is Running

There was no error output from the command.
```

Here you see it did run, it just failed to capture this output (what I see locally):

```
I1019 14:12:48.070810       1 merged_client_builder.go:122] Using in-cluster configuration
```

This is logged after the "Waiting for pod" line and before the "In project test" line.